lib/src/navbar_swipeable_utls.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+const double kDragAreaTop = 50;
+const double kDragAreaWidth = 50;
+const double kOpacityWhenSwipeable = 0.5;
+const double kDragAreaHeightFactor = 0.8;
```

Comment:
These should be hardcoded to default constants of the visible viewport dimensions 

The Defaults should be
kDragAreaTop = 0
kDragAreaWidth = NavbarRouter pages content width

Reply:
I would suggest taking a look at the Whatsapp for Android and you can see how the entire view can be scrolled

Reply:
I still don't get it 😂 then what's the point of not using PageView in the first place? I mean... I feel like this PR has strayed too far from my original idea